.github/label-actions.yml

```diff
@@ -3,10 +3,14 @@ Needs Logs:
   comment: "We need more info to debug your particular issue. If you could attach your logs to the issue (ensure no private data is in them), it would help us fix the issue much faster.
 
 1. First open the Settings page, find the `Mssql: Tracing Level` setting and change that to `All` then restart ADS.
+
 2. Open command palette (Click **View** -> **Command Palette**) and run the command `Developer: Set Log Level...` and set the level to `Trace`.
-3. Run the repro scenarios
+
+3. Run the repro scenarios.
+
 4. Collect the following logs:
 
+
```

Comment:
While you're here, can you also append on Line 40 - to restart ADS after enabling Azure logging?